linera-service/src/client.rs

```diff
@@ -968,7 +982,11 @@ fn detect_current_features() -> Vec<&'static str> {
     features
 }
 
-async fn cargo_force_build_binary(name: &'static str, package: Option<&'static str>) -> PathBuf {
+async fn cargo_force_build_binary(
+    name: &'static str,
+    package: Option<&'static str>,
+    path: &str,
```

Comment:
Would it be cleaner to use `cargo locate-project` instead of passing this in everywhere? Not sure; what do you think, @christos-h?

Reply:
Yes, I just did this so it could work, but it is for sure not the best way to do it

Reply:
@afck I'd rather not complicate things further.

Reply:
I'm not super fond of introspecting the build system in general but ok.

Reply:
Just a thought; I'm fine with either.